packages/x-date-pickers/src/MonthPicker/PickersMonth.tsx

```diff
@@ -1,106 +1,161 @@
->(Typography)(({ theme }) => ({
-  flex: '1 0 33.33%',
+const PickersYearRoot = styled('div')<{
+  ownerState: PickersMonthOwnerState;
+}>(({ ownerState }) => ({
+  flexBasis: '33.3%',
   display: 'flex',
   alignItems: 'center',
   justifyContent: 'center',
+  ...(ownerState?.wrapperVariant === 'desktop' && {
+    flexBasis: '25%',
+  }),
+}));
+
+const PickersYearButton = styled('button')<{
+  ownerState: PickersMonthOwnerState;
+}>(({ theme }) => ({
   color: 'unset',
   backgroundColor: 'transparent',
   border: 0,
   outline: 0,
   ...theme.typography.subtitle1,
   margin: '8px 0',
   height: 36,
+  width: 72,
   borderRadius: 18,
   cursor: 'pointer',
   '&:focus, &:hover': {
     backgroundColor: alpha(theme.palette.action.active, theme.palette.action.hoverOpacity),
   },
-  '&:disabled': {
-    pointerEvents: 'none',
+  [`&.${pickersMonthClasses.disabled}`]: {
     color: theme.palette.text.secondary,
   },
-  [`&.${classes.selected}`]: {
+  [`&.${pickersMonthClasses.selected}`]: {
     color: theme.palette.primary.contrastText,
     backgroundColor: theme.palette.primary.main,
     '&:focus, &:hover': {
       backgroundColor: theme.palette.primary.dark,
     },
   },
-})) as typeof Typography;
+}));
 
-const noop = () => {};
 /**
  * @ignore - do not document.
  */
-export const PickersMonth: React.FC<MonthProps> = (props) => {
+const PickersMonthRaw = (props: PickersMonthProps) => {
   const {
+    autoFocus,
+    children,
     disabled,
-    onSelect,
     selected,
     value,
     tabIndex,
-    hasFocus,
-    onFocus = noop,
-    onBlur = noop,
+    onClick,
+    onKeyDown,
+    onFocus,
+    onBlur,
     ...other
   } = props;
 
-  const handleSelection = () => {
-    onSelect(value);
-  };
+  const wrapperVariant = React.useContext(WrapperVariantContext);
 
   const ref = React.useRef<HTMLButtonElement>(null);
   useEnhancedEffect(() => {
-    if (hasFocus) {
+    if (autoFocus) {
       ref.current?.focus();
     }
-  }, [hasFocus]);
+  }, [autoFocus]);
+
+  const ownerState = { ...props, wrapperVariant };
+
+  const classes = useUtilityClasses(ownerState);
 
   return (
-    <PickersMonthRoot
-      ref={ref}
-      data-mui-test="month"
-      component="button"
-      type="button"
-      className={clsx(classes.root, {
-        [classes.selected]: selected,
-      })}
-      tabIndex={tabIndex}
-      onClick={handleSelection}
-      onKeyDown={onSpaceOrEnter(handleSelection)}
-      color={selected ? 'primary' : undefined}
-      variant={selected ? 'h5' : 'subtitle1'}
-      disabled={disabled}
-      onFocus={(event: React.FocusEvent) => onFocus(event, value)}
-      onBlur={(event: React.FocusEvent) => onBlur(event, value)}
-      {...other}
-    />
+    <PickersYearRoot data-mui-test="month" className={classes.root} ownerState={ownerState}>
+      <PickersYearButton
+        ref={ref}
+        disabled={disabled}
+        type="button"
+        data-mui-test={`month-${value}`}
```

Comment:
I don't see why adding it. aria label should allow to find the correct month. And if not, it means we have a11y problem
```suggestion
```

Reply:
If it does not break tests yes

Reply:
I removed it and all the test are passing :+1: